app/models/study.rb

```diff
@@ -10,6 +10,11 @@ def self.current_interventional
     self.interventional and self.current
   end
 
+  def self.try(nct_id='NCT00023673')
```

Comment:
The method is a quick way to create the study via a console session.  Just for testing.  I'll remove it.